### ISSUE TITLE
[Meteor 1.8.1] Unable to run in iOS with Xcode 10

### ISSUE THREAD

**Issue by J4si3k** [meteor/meteor#10673]
I can't run the meteor application on the ios emulator/simulator, during compilation I get the message:
`  TypeError: Cannot read property 'toLowerCase' of undefined
   at /Users/janusz/Desktop/iostest/.meteor/local/cordova-build/platforms/ios/cordova/lib/list-emulator-build-targets:54:45`

This is my first fresh xcode installation and I have no idea what the problem may be. additionally I download the ios 12.2 image and started the emulator for the test, so it looks like the emulator is working. I also tried `meteor reset`. Android emulator works fine. 

my kit:
- Meteor 1.8.1
- OS: macOS Mojave 10.14.6
- XCode Version 10.3 (10G8) (first tried on 11 beta)
- cordova 9.0.0 (cordova-lib@9.0.1)

**Comment by Nauzer** [comment]
Try this:
meteor update --release 1.8.1-issue-10516.0

Background:
https://github.com/meteor/meteor/issues/10516

**Comment by J4si3k** [comment]
@Nauzer unfortunately it doesn't work for me. I did the same as in this [comment](https://github.com/meteor/meteor/issues/10516#issuecomment-481418106) but without success

**Comment by Nauzer** [comment]
@J4si3k I haven't been able to try Xcode 10.3 yet. Only 11 beta 2 and 10.2.1 and those work both fine. Maybe give it a try?

I use: cordova-plugin-meteor-webapp@1.7.0

I always do the following steps before my builds to avoid any issues:
meteor reset
rm -rf node_modules
meteor npm install
meteor remove-platform ios && meteor add-platform ios

**Comment by J4si3k** [comment]
I tried. with and without plugin. different versions of xCode. it still doesn't work. maybe i didn't do something? I certainly followed the instructions in  [this guide](https://guide.meteor.com/mobile.html#installing-prerequisites-ios). This is my first xcode installation.

**Comment by Rrrom** [comment]
The same problem. 

Try with Xcode 10.3 and with Xcode 11.0 beta 4 --

nothing help to me

**Comment by WolfieZero** [comment]
I'm getting this issue whilst running through the [Meteor React tutorial](https://www.meteor.com/tutorials/react/running-on-mobile)

- macOS 10.15.2
- Xcode 11.3
- Meteor 1.8.2

**Comment by renanccastro** [comment]
Hey guys, this is fixed on this PR: https://github.com/meteor/meteor/pull/10810, but not released yet. In the meantime, you can use the `meteor run ios-device` command and run the simulator through xcode as described here https://github.com/meteor/meteor/pull/10810#issuecomment-568186020. 

Any problems, let me know.

**Comment by kakadais** [comment]
Hey gys.
Isn't there any update?

I feel like MDG is not supporting Cordova anymore..

**Comment by milanzigmond** [comment]
> Hey gys.
> Isn't there any update?

+1

**Comment by filipenevola** [comment]
Hi @kakadais and @milanzigmond we are going to still support Cordova and we are working hard on it, check this PR https://github.com/meteor/meteor/pull/10861 for Meteor 1.10 that already have 7 betas.

If you could give it a shot and provide feedback in the PR it would be amazing. Thanks

**Comment by georgiosd** [comment]
This is how I have resolved this with 1.8.1:

Trigger the build and let it fail with the `TypeError: Cannot read property 'toLowerCase' of undefined` error

Then do:
```
cd .meteor/local/cordova-build/platforms/ios/cordova/
meteor npm install ios-sim@latest
cd ../../../../../../
nano .meteor/local/cordova-build/platforms/ios/cordova/lib/list-emulator-build-targets
```
- Go to line 54 (Cmd/Ctrl+WT 54 Enter)
- Comment out the line 54 `device.availability.toLowerCase().indexOf('unavailable') < 0) {`
- Type instead `device.isAvailable === true) {`
- Save and exit (Cmd/Ctrl+O Enter, Cmd/Ctrl+X)

Build again, cheer.
